### ISSUE TITLE
elemPos / elemSize triggering BadJSON errors

### ISSUE THREAD

**Issue by JoeMShanahan** [kallisti-dev/hs-webdriver#131]
Using:
  * Chrome (Version 62.0.3202.18 (Official Build) beta (64-bit))
  * chromedriver (version 2.32)
  * selenium-server-standalone-3.5.3

At some point, an update of Chrome forced me to update Selenium and chromedriver as `click` no longer forced the browser window to scroll to the correct place leading to unfixable 'attempted to click element at x,y but couldn't' warnings.

Now, the functions `elemPos` and `elemSize` appear to expect integer numbers from Selenium which it no longer provides.

I'm seeing errors such as:

>uncaught exception: BadJSON (BadJSON "Int is either floating or will cause over or underflow: 694.0925903320312")

>uncaught exception: BadJSON (BadJSON "Word is either floating or will cause over or underflow: 1151.5")

The problem is that the JSON conversion expects integers whereas these values are retrieved from a Javascript object where all numbers are floats.

I was able to work around this with the following, using internal `hs-webdriver` functions where necessary:

```haskell
elemPosFloat :: WebDriver wd => Element -> wd (Float, Float)
elemPosFloat e = doElemCommand methodGet e "/location" Null >>= parsePair "x" "y" "elemPos"

elemPos' :: WebDriver wd => Element -> wd (Int, Int)
elemPos' e = both round <$> elemPosFloat e
  where
  both f (x, y) = (f x, f y)
```
This isn't ideal. Can we handle floats internally (or introduce functions which return floating numbers) within the library? This 'fix' feels hacky and brittle.

**Comment by kallisti-dev** [comment]
I think the best approach would be to change the API of those functions to
return floats and schedule it for a major release. if the API allows
floating point numbers we should expose that capability to the user
directly.

**Comment by JoeMShanahan** [comment]
Shall I make a pull request?

**Comment by kallisti-dev** [comment]
If you could.

**Comment by JoeMShanahan** [comment]
Done. Would be nice to get this in before lts gets bumped for ghc 8.2, if you think that's possible!

**Comment by kallisti-dev** [comment]
Fixed by #133   Also, I haven't been able to research what needs to be changed for a 8.2 upgrade, if anything. I try to maintain backwards compatibility with the last 3-4 major GHC releases.

**Comment by JoeMShanahan** [comment]
I believe this is dealt with, closing.
